packages/@dcl/ecs/src/composite/instance.ts

```diff
@@ -6,16 +6,23 @@ import { ComponentDefinition, IEngine, LastWriteWinElementSetComponentDefinition
 import { Schemas } from '../schemas'
 import { ReadWriteByteBuffer } from '../serialization/ByteBuffer'
 import { getCompositeRootComponent } from './components'
-import { ComponentData, Composite, CompositeComponent, CompositeProvider } from './types'
+import { ComponentData, CompositeComponent, CompositeDefinition } from './proto/gen/composite.gen'
```

Comment:
why not use the types defined in the Composite namespace ? They are called the same

Reply:
because there would be a cyclic dependency.
both `instance.ts` and `index.ts` (which holds the namespace) use the `proto/gen/composite.gen`